### PR TITLE
Initial setup of django

### PR DESCRIPTION
**Pull request by studioj**
added first django files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -265,4 +265,6 @@ env.bak/
 venv.bak/
 pip-selfcheck.json
 
-# End of https://www.gitignore.io/api/venv,django,python,pycharm+all
\ No newline at end of file
+# End of https://www.gitignore.io/api/venv,django,python,pycharm+all
+
+db.sqlite3
```
Comment (Jozefneefs) [inline .gitignore]: end of line missing

Comment (Jozefneefs) [reply .gitignore]: In line 270... add

Comment (studioj) [reply .gitignore]: ah ok

**Review by Jozefneefs — CHANGES_REQUESTED** [review]
OK2

File: .gitignore (hunk 1 above)
Comment (studioj) [reply .gitignore]: it was actually duplicated

**Review by Jozefneefs — APPROVED** [review]
ok3

**Pull request merged**